scripts/service-installer.sh

```diff
@@ -43,7 +43,7 @@ fi
 
 
 for service in systemd/*.service; do
-	sed "s:/home/pi/GassistPi:${repo_path}:g;s:saved-project-id:${projid}:g;s:saved-model-id:${modelid}:g;s:pi:${GIT_OWNER}:g" "$service" \
+	sed "s:/home/pi/GassistPi:${repo_path}:g;s:saved-project-id:${project_id}:g;s:saved-model-id:${device_model_id}:g;s:pi:${GIT_OWNER}:g" "$service" \
```

Comment:
As the service file does not have the model id and project id variables, this may be invalid/not required.

Reply:
I didn't update the pushbutton.py to parse the args from gassistant-credentials.info (as i said in my other comment i can't test this on my pi at the moment) that's why this line is needed for the gassistpi-push-button.service, once i update pushbutton.py to parse the arguments from gassistant-credentials.info this sed expression can be removed